feature-libs/product/image-zoom/components/product-image-zoom/default-image-zoom-layout.config.ts

```diff
@@ -0,0 +1,12 @@
+import { DIALOG_TYPE, LayoutConfig } from '@spartacus/storefront';
+import { ImageZoomDialogComponent } from './image-zoom-dialog/image-zoom-dialog.component';
+
+export const defaultImageZoomLayoutConfig: LayoutConfig = {
```

Comment:
```suggestion
export const defaultProductImageZoomLayoutConfig: LayoutConfig = {
```

That's the general rule we want to do. Everything that is exported in public API should be prefixed with the library name, before the feature name.

Reply:
done